frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetailsTable.tsx

```diff
@@ -17,34 +18,58 @@ import {
   GridContainer,
   GridCell,
 } from "./ObjectDetail.styled";
+import ExternalLink from "metabase/core/components/ExternalLink";
 
 export interface DetailsTableCellProps {
   column: any;
   value: any;
+  row: any;
   isColumnName: boolean;
   settings: any;
   className?: string;
+  getExtraDataForClick?: (clicked: any) => any;
```

Comment:
The proper type for `clicked` is [`ClickObject` (link)](https://github.com/metabase/metabase/blob/f5d85e8131903976d4b4194f7f5de728b522fe9c/frontend/src/metabase-types/types/Visualization.ts#L31)